components/Headline.js

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+export default function Headline({ name }) {
+  return (
+    <StyledHeadline>
+      <StyledH1>{name}</StyledH1>
```

Comment:
```suggestion
export default function Headline({ children }) {
  return (
    <StyledHeadline>
      <StyledH1>{children}</StyledH1>
```
I recommend you to use the children prop here. The children prop in React allows you to pass and render content or components inside another component. This approach provides more flexibility because you can include any JSX content within the Headline component, making it suitable for various use cases.